_posts/2021-03-16-notes-from-community-call-4-march-2021.md

```diff
@@ -0,0 +1,72 @@
+
+### "Flavors" of the standard
+
+Another idea that was discussed was "flavors" of the standard that certain solutions and practices are perscribed in a stronger sense than the standard is today. The openess of it now gives so many choices for the implementing party it doesn't really give much direction at all. If there were some framework specific flavors that perscribed certain styles, tools and methods, it would be easier to start developing and spend less time choosing amongst these.
+
+### Generic codebase template
+
+Related to this, but also different in the approach, one could imagine a very generic codebase (which in itself doesn't do much) that meets the standard that could be used as a reference point and inspiration. This could have the most preferred of all methods and good practices in it. But it might not be the Foundation for Public Code's role to maintain it or decide what goes in it, but rather it could be a community curated resource.
+
+### Guides
+
+More guides like the one on [how to start a Standard compliant GitHub repository](https://blog.publiccode.net/news/2020/12/09/how-to-start-a-standard-compliant-repository.html) was also requested.
+
+### Vendors checking compliance
+
+Could checking standard compliance be done by vendors? Yes, that should be possible. However, only the Foundation for Public Code can certify a codebase, but as a pre-check that should be doable. Vendors might be interested in selling this service if there is large enough demand for it.
+
+### Marketing and outreach
+
+To get more people ver the first threshold, and to establish the Standard for Pubic Code as a well known concept, there were suggestions to make more marketing of it. Unfortunately we didn't have time to discuss how to do it at any length, sout it might be topic for a separate community call in the future.
+
+### Does using the standard create a lot of overhead?
+
+Lastly there was a question if complying with the Standard would slow down the work in a codebase. Our experience is that it's a bit of an investment when starting, but that it enables collaboration and increases quality in such a way that over even a short time it will be beneficial. Doing a gap analysis is also not a huge amount of work, and the earlier it is done, the quicker it goes.
+
+For civil servants, it could also be useful to have material that could be used in a business case internally to motivate the use of ths Standard. This would be handy if an organization have previous experience of standards being costly to implement to show that this might not be the case for this one.
```

Comment:
```suggestion
For civil servants, it could also be useful to have material that could be used in a business case internally to motivate the use of ths Standard. This would be handy if an organization has previous experience of standards being costly to implement to show that this might not be the case for this one.
```